### PR TITLE
NEW - Add Sentry configuration to log runtime errors

### PR DESCRIPTION
**Pull request by CCardosoDev**
Add support to have different Sentry DSN for each environment.

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -8,6 +8,11 @@ import Footer from './components/Footer';
 import Home from './components/Home';
 import Pipelines from './components/Pipelines';
 import TopMenu from './components/TopMenu';
+import { configureSentry } from './utils'
```
Comment (mattiassluis) [inline src/App.jsx]: Let's discuss on how we can make Sentry integration something that is optional for the frontend. Just like the backend we should have a way to use plugins in Katka without forcing the user to stick with a specific tech and feature stack

File: Dockerfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Full version to make sure all developers are running the same version
-FROM node:8.11.1
+FROM artifacts.kpn.org/node:8.11.1
```
[the remark this answers is not in the document]
Comment (CCardosoDev) [reply Dockerfile]: Please refer to https://docs.npmjs.com/files/package-lock.json for excluding it from version control.

[the remark this answers is not in the document]
Comment (mattiassluis) [reply Dockerfile]: Ah my bad :(

File: Dockerfile [hunk 3]
```diff
@@ -21,8 +21,12 @@ RUN npm run build
 # Install `serve` to run the application
 RUN npm install -g serve
 
+# All environments have NODE_ENV as production
+# (options are development, test and production)
+ENV NODE_ENV=production
 # Start the node server
-CMD serve -s build
+# env configuration should land in build folder
+CMD npm run build:env dir='./build' && serve -s build
```
Comment (mattiassluis) [inline Dockerfile]: I think it is better to move the build to a bash script that we use as entrypoint script

File: README.md [hunk 4]
```diff
@@ -28,11 +44,28 @@ Frontend for Katka release orchestrator.
...
+4. Create a pull-request
+5. Wait for review and approval from the repository collaborators
+
+## Versioning
+
+We use SemVer 2 for versioning. For the versions available, see the tags on this
+repository.
+
+## Authors
+* *I&P DevSupport* - devsupport@kpn.com
```
Comment (mattiassluis) [inline README.md]: Perhaps rename this to KPN Technium DevSupport since the world doesn't know us

File: src/App.jsx (hunk 1 above)
Comment (mattiassluis) [reply src/App.jsx]: Discussed and agreed to pick this up later

**Pull request merged**